config/templates/cloud-config-worker

```diff
@@ -194,6 +194,29 @@ coreos:
         WantedBy=kubelet.service
 {{ end }}
 
+    - name: cfn-signal.service
+      command: start
+      content: |
+        [Unit]
+        Wants=kubelet.service docker.service
+        After=kubelet.service
```

Comment:
Great!
I guess I can borrow this idea to reduce useless error messages complaining about `systectl is-active flanneld.service` while kubelet.service is getting started.
Thanks 👍